packages/devtools_server/lib/src/server.dart

```diff
@@ -167,7 +167,9 @@ Future<void> registerLaunchDevToolsService(
     final VmService service = await _connectToVmService(uri);
 
     service.registerServiceCallback(launchDevToolsService, (request) async {
-      final url = '$devToolsUrl/url=${Uri.encodeComponent(uri.toString())}#';
+      // TODO(kenzie): modify this to append arguments (i.e. theme=dark). This
+      // likely will require passing in args.
+      final url = '$devToolsUrl/?url=${Uri.encodeComponent(uri.toString())}#';
```

Comment:
nit: why are we adding a '#' at the end of the url?

Reply:
removed